cli/repl.rs

```diff
@@ -29,18 +32,70 @@ impl Validator for Helper {
   }
 }
 
+async fn post_message_and_poll(
+  session: &mut InspectorSession,
+  method: String,
+  params: Option<Value>,
+  worker: &mut Worker,
```

Comment:
nitpick, make it first or second argument

Reply:
All the arguments are selected here but I'll assume you mean the worker 🙃

Reply:
It's a (bottom) one line selection unless it says `Comments on line x to y`.